Packs/CortexVulnerabilityManagement/Playbooks/Cortex_VM_-_ServiceNow_CMDB_README.md

```diff
@@ -6,35 +6,38 @@ This playbook uses the following sub-playbooks, integrations, and scripts.
 
 ### Sub-playbooks
 
-ServiceNow CMDB Search
+* ServiceNow CMDB Search
```

Comment:
When there is only one item, do not apply a bullet. This is true for lines 9 - 21.  Remove the bullet.

Reply:
again, this was created by SDK